app/models/concerns/episode_media.rb

```diff
@@ -162,11 +167,11 @@ def media=(files)
 
     # infer episode medium
     current = contents.reject(&:marked_for_destruction?)
-    unless medium_uncut?
-      if current.all?(&:audio?)
-        self.medium = "audio"
+    if !medium_uncut? && current.present?
+      self.medium = if current.all?(&:audio?)
+        "audio"
```

Comment:
But this _is_ a functional change?  Seems like it won't set medium to `audio` anymore when you have 0 contents.

And it will set medium to nil if you send a mix of contents.